src/config/dynamic-theme-fixes.config

```diff
@@ -21816,7 +21816,7 @@ img[alt="audio speaker icon"]
 img[src*="Wiktionary-logo"]
 
 CSS
-.mwe-popups-discreet > svg,
+.mwe-popups-discreet > img,
```

Comment:
This will invert every popup image, that isn't correct behavior.

Reply:
But this isn't an image inversion... All this does is to make the background color of the images in the pop-up box white, so that it matches the behaviour of the original webpage. That way it shouldn't cause any problems as long as the images themselves aren't inverted (and that part is already taken care of).

Reply:
Ah oops, sorry. it looked like this was the invert section.